index.bs

```diff
@@ -2272,6 +2264,103 @@ Authenticators:
+    by entering a PIN or scanning a fingerprint.
+
+    In this use case the [=authenticator=] is [=something you have=] which also verifies [=something you know=] and/or [=something
+    you are=], depending on the particular [=authenticator=]. This use case is only available with [=authenticators=] that support
+    [=user verification=].
+
+- In the <dfn>username-less use case</dfn>, the user initiates an [=authentication ceremony=] without any initial input. The
+    [=[RP]=] then asks for an [=assertion=] by any [=public key credential|credential=]. The user picks a [=public key
+    credential|credential=] from those stored in the [=authenticator=], and the [=user handle=] returned in the [=assertion=] is
+    used to identify the user. The [=[RP]=] might then ask the user for a password, or log the user in using only the verified
+    [=assertion=].
+
+    In this use case the [=authenticator=] is [=something you have=] and the password, if used, is [=something you know=]. This
+    use case is only available with [=authenticators=] that support [=client-side-resident credential private keys=].
+
+- In the <dfn>single-step use case</dfn>, the user initiates an [=authentication ceremony=] without any initial input. The
+    [=[RP]=] then asks for an [=assertion=] with [=user verification=] by any [=public key credential|credential=]. The user picks
+    a [=public key credential|credential=] from those stored in the [=authenticator=] and performs [=user verification=], e.g., by
+    entering a PIN or scanning a fingerprint, and the [=user handle=] returned in the [=assertion=] is used to identify the user.
+
+    In this use case the authenticator is [=something you have=] which also verifies [=something you know=] and/or [=something you
+    are=], depending on the particular [=authenticator=]. This use case is only available with [=authenticators=] that support
+    both [=user verification=] and [=client-side-resident credential private keys=].
+
+
+### <dfn>Attachment Modality</dfn> ### {#sctn-authenticator-attachment-modality}
+
+[=Clients=] can communicate with [=authenticators=] using a variety of mechanisms. For example, a [=client=] MAY use a
+platform-specific API to communicate with an [=authenticator=] which is physically bound to a platform. On the other hand, a
+[=client=] can use a variety of standardized cross-platform transport protocols such as Bluetooth (see [[#transport]]) to discover
+and communicate with cross-platform attached [=authenticators=]. We refer to [=authenticators=] that are part of the [=client=]'s
+platform as <dfn>platform authenticators</dfn>, while those that are reachable via cross-platform transport protocols are referred
+to as <dfn>roaming authenticators</dfn>.
+
+<ul>
+    <li>A [=platform authenticator=] is attached using a platform-specific transport, and is usually not removable from the
```

Comment:
In FIDO world the term "Bound Authenticator" is used for that.

Reply:
This is not a new term, I just broke this section out from the [Authenticator Attachment enumeration](https://w3c.github.io/webauthn/#attachment) section.